content/en/docs/concepts/storage/volumes.md

```diff
@@ -352,7 +352,9 @@ third party storage driver instead.
 ### gitRepo (deprecated) {#gitrepo}
 
 {{< warning >}}
-The `gitRepo` volume type is deprecated.
+The `gitRepo` volume plugin has been disabled by default.
+
+The plugin can be enabled by setting feature-gate GitRepoVolumeDriver=true.
```

Comment:
```suggestion
You can use this deprecated storage plugin in your cluster if you explicitly enable the `GitRepoVolumeDriver`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
```

I would move this sentence to be the first line after the warnings. We want people to read the warnings.

Reply:
Done. Thanks!